generateARMTs.sh

```diff
@@ -1,7 +1,7 @@
 source .env
-node utilities/index.js $REDCAP_API_URL $DA_REDCAP_API_TOKEN  aRMT $DA_LANG
-node utilities/index.js $REDCAP_API_URL $DE_REDCAP_API_TOKEN  aRMT $DE_LANG
-node utilities/index.js $REDCAP_API_URL $EN_REDCAP_API_TOKEN  aRMT ''        # only seems to work with a literal for EN_LANG
-node utilities/index.js $REDCAP_API_URL $ES_REDCAP_API_TOKEN  aRMT $ES_LANG
-node utilities/index.js $REDCAP_API_URL $IT_REDCAP_API_TOKEN  aRMT $IT_LANG
-node utilities/index.js $REDCAP_API_URL $NL_REDCAP_API_TOKEN  aRMT $NL_LANG
+node utilities/index.js $REDCAP_API_URL $DA_REDCAP_API_TOKEN  aRMT $DA_LANG $1
```

Comment:
perhaps instead of `$1` add $FORM_NAMES to `utilities/_.env-template` and you can keep the same convention, handy to be able to keep these in a env file to source when needed

Reply:
Thanks, made the changes.